indexCorrected.js

```diff
@@ -16,4 +16,8 @@ app.get("/api/misfortune", getMisfortune);
 app.post("/api/make", createMake);
 app.delete("/api/make/:index", deleteMake);
 
+app.get("/", (req, res) => {
+  res.sendFile(__dirname + "/indexCorrected.html")
+})
+
```

Comment:
This here is super important without it you won't see your html page populate. What this `app.get()` does is every time the url `localhost:4000/` is called it calls this function. The function then sends the html page back to the browser so that it can be displayed.